src/components/StockSystem/SymbolsList.tsx

```diff
@@ -409,7 +409,7 @@ const SymbolsList = () => {
       <Box>
 
         <DataGrid sx={{
-          height: '94vh',
+          height: 'calc(100dvh - 48px)',
```

Comment:
This should be on the top container and then its childrens should use %.
also, where 48px is coming from? i dont think its the best practice. please find the best practice.